src/priceClient/adapters/across.ts

```diff
@@ -0,0 +1,60 @@
+import assert from "assert";
```

Comment:
nit: might be more appropriate to call this file across-api or something. Just so it's more obvious that we're getting the prices from the _api_.

Reply:
Implemented here: https://github.com/across-protocol/sdk-v2/pull/98/commits/1eb2eda88fdc8ccffa543e78e03d8e1b7bb3baae